tests/post/steps/conftest.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+from configparser import ConfigParser
+from itertools import chain
+
+import pytest
+from pytest_bdd import(
+    given,
+    then,
+    when,
+)
+
+
+# Pytest fixtures
+@pytest.fixture(scope="session")
+def version():
+    """Fixture that read metalk8s version.
+
+    Version is extracted from _build/root/product.txt
+    """
+    version_file = "_build/root/product.txt"
+    parser = ConfigParser()
+
+    with open(version_file) as stream:
+        # add a section header for configparser
+        stream = chain(("[top]",), stream)
+        parser.readfp(stream)
+
+    return {k: v for k, v in parser.items('top')}
```

Comment:
parser.items() is a dict ? why not returning it directly ?

Reply:
It is a list. Something like that:
```
[('name', 'MetalK8s'), ('version', '2.0.0-dev'), ('short_version', '2.0'), ('git', '2.0.0-alpha1-7-gbb0bf9c'), ('development_release', '1'), ('build_timestamp', '2019-03-13T17:09:47Z'), ('build_host', 'duhamel-Latitude-E7450')]
```